cmd/kured/main.go

```diff
@@ -96,7 +96,7 @@ func main() {
 		Run:    root}
 
 	rootCmd.PersistentFlags().BoolVar(&forceReboot, "force-reboot", false,
-		"force a reboot even if the drain is still running (default: false)")
+		"force a reboot even if the drain fails or times out (default: false)")
```

Comment:
if you would like this clarification to be included in the next release (1.7.0), it's probably better to add it into another PR.

Reply:
I'm not overly concerned, I just noticed it was slightly incorrect when doing the helm chart changes and wanted to correct it.